### ISSUE TITLE
Context menu for DB objects

### ISSUE THREAD

**Issue by GrigoriyMikhalkin** [beekeeper-studio/beekeeper-studio#246]
Hi! Is there any plans to add context menu for database objects similar to one that pgadmin has?
![pgadmin_context_meny](https://user-images.githubusercontent.com/3637857/87478153-94c33000-c629-11ea-8e06-de7d3efd82e8.png)

Maybe one with less options than in pgadmin. E.g. it could provide options that would generate templates for dropping/creating table, viewing data and other often used operations.

**Comment by rathboma** [comment]
Yes, we have plans to do this! I'll keep around this ticket so you can see when we finally get it done. :-)

**Comment by UtechtDustin** [comment]
ref #275
